tests/Unit/Lti/LoadBalancer/UsernameLtiInstanceLoadBalancerTest.php

```diff
@@ -43,13 +44,19 @@ protected function setUp(): void
     {
         parent::setUp();
 
+        $ltiInstanceId1 = new UuidV4('00000000-0000-4000-0000-000000000001');
```

Comment:
Suggestion: you could avoid this variables

Reply:
Fixed